### PR TITLE
Add termUnit property

### PR DESCRIPTION
**Pull request by OvervCW**
Adds the missing `termUnit` property for the `term` object within `Subscription`.

### CODE REVIEW

File: Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json [hunk 1]
```diff
@@ -943,6 +943,9 @@
               "endDate": {
                 "type": "string",
                 "format": "date-time"
+              },
+              "termUnit": {
```
Comment (rokf) [inline Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json:947]: Couldn't this be an enum with `P1M` and `P1Y`.

Comment (OvervCW) [reply Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json]: Yeah, I just noticed that the definition is inconsistent with the other `termUnit` definitions, fixed.

Comment (santhoshb-msft) [reply Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json]: Thank you, already updated !

**Comment by santhoshb-msft** [comment]
Thank you already updated!

**Comment by OvervCW** [comment]
@santhoshb-msft I don't see it. The termUnit property is still missing in the latest commit on the main branch:

https://github.com/microsoft/commercial-marketplace-openapi/blob/2c5adc39ad5ea5aa05b22ef61bef481e7d8d8c8a/Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json#L936-L948

**Pull request closed without merging**